sys/kos/init.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+//go:generate go run fidlgen/main.go
+
+package kos
+
+import (
+	"github.com/google/syzkaller/prog"
+	"github.com/google/syzkaller/sys/targets"
+)
+
+func InitTarget(target *prog.Target) {
+	target.MakeDataMmap = targets.MakeSyzMmap(target)
```

Comment:
Hey Manasse,

for InitTarget, I used the same one as the one used for Linux, because KOS also has mmap, just as linux:

func InitTarget(target *prog.Target) {
	target.MakeDataMmap = targets.MakePosixMmap(target, true, true)
}

Reply:
Oh, I see. I think we can keep it for now, since it is Go code and all the Go code will run on the host. We will see when executor is ready how It goes.